portfolio/components/ui/Figures/LineItemDecoderVisualization/LineItemDecoderVisualization.module.css

```
@@ -0,0 +1,223 @@
+  text-transform: uppercase;
+  letter-spacing: 0.05em;
+  color: var(--text-color);
+  opacity: 0.6;
+  white-space: nowrap;
+}
+
+.verdictValue {
+  font-size: 1.05rem;
+  font-weight: 700;
+  color: var(--text-color);
+  font-family: "Monaco", "Menlo", "Consolas", monospace;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Monaco" (font-family-name-quotes)

(font-family-name-quotes)